### PR TITLE
Add Wine Dependency information to Readme

### PR DESCRIPTION
**Pull request by termdisc**
For non-flatpaks, refer to Wine dependencies.

<--- Put the description here --->

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Comment by Gamertech** [comment]
Think it's because of the extra brackets around the link. Compare to the link directly below your commit.

File: README.md [hunk 1]
```diff
@@ -146,6 +146,10 @@ Thanks to the community, Heroic was translated to almost 40 different languages
 
 Heroic is available on Flathub, so you should be able to easily install it on most distros with Software Centers (Pop!\_Shop, Discover, etc.)
 
+#### Non-Flatpak Installations
```
Comment (CommandMC) [inline README.md:149]: There's a trailing space here (that's why Prettier is not passing)
```suggestion
#### Non-Flatpak Installations
```

**Review by CommandMC — CHANGES_REQUESTED** [review]
I'm not sure how we want to proceed, with this & #3687 doing the same thing

**Comment by flavioislima** [comment]
This information was added on another PR.
thanks for the contribution.

**Pull request closed without merging**